doc/mgr/dashboard.rst

```diff
@@ -133,6 +133,18 @@ dashboard.
 
 .. _dashboard-landing-page-status:
 
+Switch Dashboard Landing Page
```

Comment:
BTW we had quite a verbose description of [the previous Landing Page](https://docs.ceph.com/en/quincy/mgr/dashboard/#overview-of-the-dashboard-landing-page), why not putting here what you did for the [Ceph Blog](https://landing-page-blog.ceph.io/en/news/blog/2023/landing-page/) (in a less formal/merely descriptive style), explaining each card and reusing the screenshots?

Additionally, I find this screenshot a bit dull compared to the one in the blog:
![image](https://user-images.githubusercontent.com/37327689/224303768-33b912ed-1b97-46c4-994c-c2617758b30c.png)
![image](https://user-images.githubusercontent.com/37327689/224303793-ddc5678f-4de3-40ff-bda3-470eeb9d9854.png)